### DISCUSSION TITLE
Boot loop on LOS 20 - Redmi Note 9 (merlinx)

### DISCUSSION THREAD

**Discussion by Weissnix4711** [bmax121/APatch#524] [Q&A] [answered]
Kernel is patched successfully by the app, but on flashing and reboot, ends in boot loop.

LOS: `lineage-20.0-20240618-nightly-merlinx-signed.zip` https://download.lineageos.org/devices/merlinx/builds
Apatch app: https://github.com/bmax121/APatch/releases/tag/10763

```
merlinx:/ $ zcat /proc/config.gz | grep -w CONFIG_KALLSYMS
CONFIG_KALLSYMS=y
merlinx:/ $ zcat /proc/config.gz | grep -w CONFIG_KALLSYMS_ALL                                                               
# CONFIG_KALLSYMS_ALL is not set
```

Does it matter CONFIG_KALLSYMS_ALL is unset?

**Comment by marat2509** [comment, marked as answer]
Yes. For correct and stable operation APatch support is required `CONFIG_KALLSYMS_ALL` in the kernel
